### ISSUE TITLE
Different Jobs contains different versions of library Zebrunner-CE@1.3,1,1,1,4

### ISSUE THREAD

**Issue by okamara** [zebrunner/jenkins-master#338]
Verified on master 
Steps:
1. git clone --recurse-submodule https://github.com/zebrunner/zebrunner.git
2. Set up zebrunner
3. Register organization
4. Open Configuration for different jobs and verify version of library
Actual result:
Launcher -> @Library(‘Zebrunner-CE@1.3’)

@Library(‘Zebrunner-CE@1.4’)
-> com.zebrunner.jenkins.pipeline.Repository;

@Library(‘Zebrunner-CE@1.1’)
->com.zebrunner.jenkins.pipeline.Organization

Expected result: @Library(‘Zebrunner-CE@1.4’)

**Comment by vdelendik** [comment]
fixed in 1.5-beta1 jenkins-master. Already added as part of zebunner develop branch.
After setup verify 
1. that all jobs in Management_Jobs folder and RegisterRepository and launcher jobs in root folder have master branch for Zebrunner-CE pipeline, i.e. pipelice scipt 1st line like below:
```
@Library('Zebrunner-CE@master')
import com.zebrunner.jenkins.pipeline.runner.maven.TestNG;
new com.zebrunner.jenkins.pipeline.runner.maven.TestNG(this).runJob()
```

2. also verify that there is no ciStart token for all above default jobs

**Comment by okamara** [comment]
Verified on dev. Fixed.
